model/screenshot.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+type Screenshot struct {
+	ScreenshotID   int64    `json:"screenshot_id,omitempty"`
+	Title          string   `json:"title,omitempty"`
+	Description    string   `json:"description,omitempty"`
+	ScreenshotTags []string `json:"screenshot_tags,omitempty"`
+	URL            string   `json:"url,omitempty"`
+}
```

Comment:
The documentation states that we should expect a `key_ids` array. Possibly of type `[]int64`. 
The fields `width` and `height` are missing as well. 

Will you add them?

Reply:
`keyIDs` should be exported, ie. `KeyIDs`